test/functional/eval/reltime_spec.lua

```diff
@@ -0,0 +1,36 @@
+local helpers = require('test.functional.helpers')
+local clear, eq, ok = helpers.clear,  helpers.eq, helpers.ok
+local neq, execute, funcs  = helpers.neq, helpers.execute, helpers.funcs
+local reltime, reltimestr, reltimefloat = funcs.reltime, funcs.reltimestr, funcs.reltimefloat
+
+describe('Basic functionality of reltimestr and reltimefloat', function()
```

Comment:
describe('reltimestr(), reltimefloat()', function()

Try to be brief yet precise in `describe()` and each nested `it()`: the titles are concatenated to make a single line. And it's a good idea in general.